<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lib_networkManage</class>
 <widget class="QWidget" name="lib_networkManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lib_networkManage{background-color: rgb(248, 248, 255);}</string>
  </property>
  <widget class="QRadioButton" name="raBut_hotspot">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>87</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/prefix2/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/prefix2/LCD_CS351/logo/Radio_check.png);
}</string>
   </property>
   <property name="text">
    <string>热点</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="raBut_wifi">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>560</y>
     <width>87</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/prefix2/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/prefix2/LCD_CS351/logo/Radio_check.png);
}</string>
   </property>
   <property name="text">
    <string>无线</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>网卡模式:</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackedWidget{background-color: rgb(248, 248, 255);}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>网络搜索提示</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="psk">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>520</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SSID">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>490</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>360</width>
       <height>430</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{
font: 12pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;
}
QListWidget::item{/*子项样式*/
height:35px;/*高度*/
}
QListWidget::Item:selected {/*选中项样式*/
background-color:rgb(51, 153, 255);
color: rgb(255, 255, 255);
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>无线名称:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>无线密码:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>520</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/view_off.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>热点密码:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hotspot_psk">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_viewPsk">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/view_off.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>热点名称:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hotspotName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>热点IP:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hotspot_ip">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hotspot_ip_head">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>195</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>IP分配段:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hotspot_ip_end">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>开启时间:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="openTime">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_save">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>490</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/prefix2/LCD_CS351/logo/saves.bmp);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="but_cancel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>490</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/prefix2/LCD_CS351/logo/cancel.bmp);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>小时</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>label_20</zorder>
  <zorder>raBut_hotspot</zorder>
  <zorder>raBut_wifi</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
